### DISCUSSION TITLE
Widget showcase page is live

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#810] [Announcements]
Hi Astroiders,

Astroid with outstanding utility system helps you easily build your website quickly. I'm happy to announce that the widget showcase page is live.

Link: https://astroidframe.work/widgets

Let's check it and give me the feedbacks

Best Regards,
Sonny

**Comment by Chacapamac** [comment]
Sorry, Sonny to be not too responsive these days. I need to do a major upgrade of my old 2011 iMac to continue testing Astroid.

I’m so impressed with your work on these widgets and Astroid in general, fantastic. Also, integrating the widgets examples and documentation on the Astroid Website is so cool. I’m sure you know that you have built the ONLY framework that should be used by Joomla. BRAVO!. 

I will look deeper at each Widget and give you feedback. Without the ability to test directly on my computer is really bugging me. I will fix this when our Canadian winter keeps me inside. Also, I need to take a look at the changes you made on the Astroid website, looks amazing. 🇨🇦 ⛄️

**Comment by Dnsftl** [comment]
Review of the Astroid Framework Widgets Page'

The Astroid Framework widgets page provides a clear and practical overview of the available widgets, designed to support developers in their implementation.

Strengths:

- Interactivity: The page includes active examples, allowing users to directly experience the functionality and appearance of each widget. This facilitates decision-making for implementation.
- Responsiveness: The website is optimized for various screen sizes, ensuring a consistent and user-friendly experience on both desktop and mobile devices.
- Documentation: Extensive links to detailed documentation provide clear installation instructions and configuration options, enhancing the understanding of the widgets.

Overall, the Astroid Framework widgets page is a valuable resource for developers looking to efficiently work with various widgets.
